### PR TITLE
Added option to skip pretest build

### PR DESCRIPTION
**Pull request by matthewellis**
Similar to https://github.com/fastlane/scan/pull/62 we have found that this extra build step is not needed for us but i'm aware that many users may be relying on this for some build artifacts so i propose adding an option that would allow this to be skipped without changing the default behaviour.

### CODE REVIEW

File: lib/scan/options.rb [hunk 1]
```diff
@@ -50,6 +50,11 @@ def self.available_options
                                      description: "Should generate code coverage (Xcode 7 only)?",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_pretest_build,
```
Comment (KrauseFx) [inline lib/scan/options.rb]: Let's rename this to `skip_build`

**Comment by KrauseFx** [comment]
Great idea, I added a comment before it's ready to be merged. Please also squash the commits and let me know when you updated the PR :+1:

**Comment by matthewellis** [comment]
Hey thanks for the feedback, i have made the change to the key but for some reason travis is now failing the build but it seems to be an issue with travis from looking at the logs.

**Comment by KrauseFx** [comment]
Yah, seems like a Travis issue, don't worry about it

**Comment by KrauseFx** [comment]
Looks great, thanks for updating the PR, this will be in the next release :+1:

**Pull request merged**